### ISSUE TITLE
Chibi toy

### ISSUE THREAD

**Issue by Amymcdougall** [lockedata/merchideas#15]
Because I want the kids to have a chibi Steph or chibi Amy to play with 😂😂 

You never know they could be popular?

**Comment by stephlocke** [comment]
I had #12 in mind but we could get plushy things

**Comment by Amymcdougall** [comment]
Yea that's the kind of things I mean. I thought those were those literal bobble heads 😂 with tiny body and the head that giggle.

**Comment by stephlocke** [comment]
yeah - well I don't expect them to make noise - you know how much I hate noise!

**Comment by Amymcdougall** [comment]
Aha, we will have to look into places that do plush toys
